apps/mobile-mzima-client/src/app/profile/information/information.page.ts

```diff
@@ -136,8 +163,14 @@ export class InformationPage {
       });
   }
 
-  public handlePhotoChange(event: boolean): void {
-    this.isPhotoChanged = event;
+  public handlePhotoSelected(event: { key: string; caption: string }): void {
+    if (this.selectedFileKey !== event.key || this.selectedCaption !== event.caption) {
+      this.selectedFileKey = event.key;
```

Comment:
I think `this.selectedFileKey` and `event.key` will always be the same, so this will never detect the new photo.

Reply:
Thats a good catch. I've just tried it rignt now. Let me see if I can introduce a new property to keep track of the last successfully uploaded key